jormungandr/src/fragment/process.rs

```diff
@@ -55,61 +55,62 @@ impl Process {
 
-                    // This interface only makes sense for messages coming from arbitrary users (like transaction, certificates),
-                    // for other message we don't want to receive them through this interface, and possibly
-                    // put them in another pool.
+                        // This interface only makes sense for messages coming from arbitrary users (like transaction, certificates),
+                        // for other message we don't want to receive them through this interface, and possibly
+                        // put them in another pool.
 
-                    let stats_counter = stats_counter.clone();
+                        let stats_counter = stats_counter.clone();
 
-                    pool.insert_and_propagate_all(origin, txs)
-                        .await
-                        .map(move |count| stats_counter.add_tx_recv_cnt(count))?;
-                }
-                TransactionMsg::RemoveTransactions(fragment_ids, status) => {
-                    tracing::debug!(
-                        "removing fragments added to block {:?}: {:?}",
-                        status,
-                        fragment_ids
-                    );
-                    pool.remove_added_to_block(fragment_ids, status, service_info.logger());
-                }
-                TransactionMsg::GetLogs(reply_handle) => {
-                    let logs = pool.logs().logs().cloned().collect();
-                    reply_handle.reply_ok(logs);
-                }
-                TransactionMsg::GetStatuses(fragment_ids, reply_handle) => {
-                    let mut statuses = HashMap::new();
-                    pool.logs().logs_by_ids(fragment_ids).into_iter().for_each(
-                        |(fragment_id, log)| {
-                            statuses.insert(fragment_id, log.status().clone());
-                        },
-                    );
-                    reply_handle.reply_ok(statuses);
-                }
-                TransactionMsg::SelectTransactions {
-                    pool_idx,
-                    ledger,
-                    block_date,
-                    ledger_params,
-                    selection_alg,
-                    reply_handle,
-                } => {
-                    let contents =
-                        pool.select(pool_idx, ledger, block_date, ledger_params, selection_alg);
-                    reply_handle.reply_ok(contents);
+                        pool.insert_and_propagate_all(origin, txs)
+                            .await
+                            .map(move |count| stats_counter.add_tx_recv_cnt(count))?;
+                    }
+                    TransactionMsg::RemoveTransactions(fragment_ids, status) => {
+                        tracing::debug!(
+                            "removing fragments added to block {:?}: {:?}",
+                            status,
+                            fragment_ids
```

Comment:
I guess this would work. But for what I check on the library the way of formatting is using [`sigils`](https://tracing.rs/tracing/#recording-fields), at least for fields. It should be ok in this case.